acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/ibft2/Ibft2BlockRewardPaymentAcceptanceTest.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.tests.acceptance.ibft2;
+
+import static java.util.Collections.singletonList;
+
+import org.hyperledger.besu.ethereum.core.Address;
+import org.hyperledger.besu.tests.acceptance.dsl.AcceptanceTestBase;
+import org.hyperledger.besu.tests.acceptance.dsl.account.Account;
+import org.hyperledger.besu.tests.acceptance.dsl.blockchain.Amount;
+import org.hyperledger.besu.tests.acceptance.dsl.node.BesuNode;
+
+import java.io.IOException;
+import java.math.BigInteger;
+import java.util.Optional;
+
+import org.junit.Test;
+
+public class Ibft2BlockRewardPaymentAcceptanceTest extends AcceptanceTestBase {
+
+  @Test
+  public void validatorsArePaidBlockReward() throws IOException {
```

Comment:
Do we need a test with multiple validators receiving a fixed reward? Though that get complicated so might not be worth it.

Reply:
Yeah, it got scary when you start saying "how do you know"? I.e. it depends on rounds, and blocks...
maybe you just wait until 2 validators have been paid (something ... anything) - no API exists in the DSL to do that, but sure we can make it work